src/main/scala/mesosphere/marathon/api/HttpTransferMetricsHandler.scala

```diff
@@ -86,17 +86,17 @@ class HttpTransferMetricsHandler(httpMetrics: HttpTransferMetrics) extends Abstr
     if (!isExcluded(request)) {
       val bytesRead = request.getContentLengthLong
```

Comment:
I just realised that I _did_ had a look on this variable when I wrote the first version of the http metrics. The `getContentLength` is just returning the value of the `Content-Length` request header 😄 

In principle, most of the HTTP clients populate this correctly, but the bottom-line is that this is still a header. That's why I went with counting the physical bytes written in the stream ¯\\\_(ツ)\_/¯ .

However, since this is just a metric and not a critical part of marathon, we are good for 99% of the cases 👍

Reply:
Updated. It uses the bytes read counter now.

Reply:
(It wasn't as obvious to find the counter for bytes read, but after some digging I found it)